### PR TITLE
Add weakdeps

### PR DESCRIPTION
**Pull request by jjgomezcadenas**
weakdeps replaces Requires.jl for newever verion of julia keeping compatibility with older versions.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -8,11 +8,22 @@ BayesHistogram = "000d9b38-65fe-4c81-bdb9-69f01f102479"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MakieCore = "20f20a25-4f0e-4fdf-b5d1-57303727442b"
 Measurements = "eff96d63-e80a-5855-80a2-b1b0885c5ab7"
+Pluto = "c3e4b0f8-55cb-11ea-2926-15256bba5781"
```
Comment (Moelf) [inline Project.toml:11]:
```suggestion

```

**Comment by Moelf** [comment]
look like some code loading didn't work?

**Comment by jjgomezcadenas** [comment]
The error is in the docs, which includes a notebook (makie_plotting.jl). Is this a Pluto notebook? I did not realize it was there. I will try and take a look see what is going on.

**Comment by Moelf** [comment]
yeah it's the notebook used to generate: https://moelf.github.io/FHist.jl/dev/notebooks/makie_plotting/

**Comment by jjgomezcadenas** [comment]
Hi, took a quick look, can't figure out what it is without a thorough investigation. All what I did was to change the Requires.jl "recipe" by the new recipe using weakdeps. Apparently it works well for Plots, but not for Makie. Could it be yet a "Pluto thing?".

**Comment by Moelf** [comment]
no worries, I will take a look!

**Pull request merged**